src/main/kotlin/io/nais/security/oauth2/TokenExchangeApp.kt

```diff
@@ -193,18 +191,10 @@ private fun ErrorObject.toGeneric(): ErrorObject =
     )
 
 internal val defaultHttpClient = HttpClient(CIO) {
-    install(JsonFeature) {
-        serializer = JacksonSerializer {
-            configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-            setSerializationInclusion(JsonInclude.Include.NON_NULL)
+    install(io.ktor.client.plugins.contentnegotiation.ContentNegotiation) {
+        jackson() {
+            configure(FAIL_ON_UNKNOWN_PROPERTIES, false)
+            setSerializationInclusion(NON_NULL)
         }
     }
 }
-
-object Jackson {
-    val defaultMapper: ObjectMapper = jacksonObjectMapper()
```

Comment:
Will there be a performance penalty by removing the singleton instance of the objectmapper and create a new instance every time?